p2p.go

```diff
@@ -213,6 +212,16 @@ func PrivateNetworkPSK(privateNetworkPSK string) Option {
 	}
 }
 
+// DHTProtocolName returns the prefix of dht protocol. MainNet uses "/iotex", while testNet uses "/iotexTestNet"
+func DHTProtocolName(chainID uint32) Option {
+	return func(cfg *Config) error {
+		if chainID != 1 {
+			cfg.DHTProtocolName = "/iotexTestNet"
```

Comment:
let's use "iotex2" for chainID = 2, "iotex7" for chainID = 7 for generic case
then in the future, we don't need to modify here if someone starts up a new sub-network